File: fizz_buzz.rb
def fizzbuzz(number)
	divisible_by_3 = number % 3 == 0
	divisible_by_5 = number % 5 == 0

	case 
	when divisible_by_3 && divisible_by_5
		puts "FizzBuzz!"
	when divisible_by_3
		puts "Fizz"
	when divisible_by_5
		puts "Buzz"
	else
		puts number
	end
end

File: fizzbuzz.js
// fizzbuzz.js
// splashinn

for (var i = 1; i <= 100; i++) {
    var expletive = '';
    if (i % 3 === 0) expletive += 'Fizz';
    if (i % 5 === 0) expletive += 'Buzz';
    console.log(expletive || i);
}
